plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -84,6 +92,59 @@ public TestingPostgreSqlServer(boolean shouldExposeFixedPorts)
         execute("CREATE SCHEMA IF NOT EXISTS tpch");
     }
 
+    private class RemoteDatabaseEventLogConsumer
+            implements Consumer<OutputFrame>
+    {
+        private boolean cancellationHit;
+
+        @Override
+        public void accept(OutputFrame outputFrame)
+        {
+            if (tracingEvents.isEmpty()) {
+                return;
+            }
+
+            buildEvent(outputFrame)
+                    .ifPresent(remoteDatabaseEvent -> tracingEvents.forEach(tracingEvent -> tracingEvent.accept(remoteDatabaseEvent)));
+        }
+
+        private Optional<RemoteDatabaseEvent> buildEvent(OutputFrame outputFrame)
```

Comment:
Is `getRemoteDatabaseEvents` now safe to remove?

Reply:
There are multiple references of the `getRemoteDatabaseEvents`, I think it's not so easy to remove the method directly.